### PR TITLE
full-docs changes for operator 1.1.0

### PR DESCRIPTION
**Pull request by abby-cyber**

### CODE REVIEW

**Review by whitewum — DISMISSED** [review]

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The following features are already available in Nebula Operator:
   
 - **Scale clusters**: Nebula Operator calls Nebula Graph's native scaling interfaces in a control loop to implement the scaling logic. You can simply perform scaling operations with YAML configurations and ensure the stability of data. For more information, see [Scale clusters with Kubectl](3.deploy-nebula-graph-cluster/3.1create-cluster-with-kubectl.md) or [Scale clusters with Helm](3.deploy-nebula-graph-cluster/3.2create-cluster-with-helm.md).
   
-- **Cluster Upgrade**: Nebula Operator supports cluster upgrading from version 2.5.x to version 2.6.x.
+- **Cluster Upgrade**: Nebula Operator supports cluster upgrading from version {{upgrade_from}} to version {{upgrade_to}}.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:21]: Wrong macro formats.
operator.upgrade_from
operator.upgrade_to

File: docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md [hunk 2]
```diff
@@ -32,13 +32,20 @@ Nebula Operator does not support the v1.x version of Nebula Graph. Nebula Operat
...
-| {{operator.release}}| 2.5.x ~ 2.6.x |
+| 1.1.0| 3.0.0 ~ 3.1.x |
+| 1.0.0| 3.0.0 ~ 3.1.x |
+| 0.9.0| 2.5.x ~ 2.6.x |
 |0.8.0|2.5.x|
 
+!!! Compatibility "Legacy version compatibility"
+
+    - The 1.x version Nebula Operator is not compatible with Nebula Graph of version below v3.x.
+    - Starting from Nebula Operator 0.9.0, logs and data are stored separately. Using Nebula Operator 0.9.0 or later versions to manage a Nebula Graph 2.5.x cluster created with Operator 0.8.0  can cause compatibility issues. You can backup the data of the Nebula Graph 2.5.x cluster and then create a 2.6.x cluster with Operator 0.9.0.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/1.introduction-to-nebula-operator.md:43]:
```suggestion
    - Starting from Nebula Operator 0.9.0, logs and data are stored separately. Using Nebula Operator 0.9.0 or later versions to manage a Nebula Graph 2.5.x cluster created with Operator 0.8.0 can cause compatibility issues. You can backup the data of the Nebula Graph 2.5.x cluster and then create a 2.6.x cluster with Operator 0.9.0.
```

**Review by foesa-yang — APPROVED** [review]

File: docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md [hunk 3]
```diff
@@ -2,23 +2,23 @@
...
-## Limits
-
-- Only Nebula Graph clusters created with Nebula Operator are supported.
+    The 1.x version Nebula Operator is not compatible with Nebula Graph of version below v3.x.
 
-- Only upgrading Nebula Graph from 2.5.x to 2.6.x is supported.
+## Limits
 
-- Upgrading clusters created via Nebula Operator of version 0.8.0 is not supported.
+- Only for Nebula Graph clusters that were created with Nebula Operator.
```
Comment (randomJoe211) [inline docs-2.0/nebula-operator/9.upgrade-nebula-cluster.md:11]: No need to use the the past tense.

**Review by randomJoe211 — DISMISSED** [review]

File: docs-2.0/20.appendix/6.eco-tool-version.md [hunk 4]
```diff
@@ -74,9 +74,13 @@ Nebula Exchange (Exchange for short) is an Apache Spark&trade application for ba
 |:---|:---|:--|
 | {{ nebula.release }}  | {{exchange.release}} | {{exchange_ent.release}} |
 
-<!-- ## Nebula Operator
+## Nebula Operator
 
-Nebula Operator (Operator for short) is a tool to automate the deployment, operation, and maintenance of Nebula Graph clusters on Kubernetes. Building upon the excellent scalability mechanism of Kubernetes, Nebula Graph introduced its operation and maintenance knowledge into the Kubernetes system, which makes Nebula Graph a real cloud-native graph database. For more information, see [What is Nebula Operator](../nebula-operator/1.introduction-to-nebula-operator.md). -->
+Nebula Operator (Operator for short) is a tool to automate the deployment, operation, and maintenance of Nebula Graph clusters on Kubernetes. Building upon the excellent scalability mechanism of Kubernetes, Nebula Graph introduced its operation and maintenance knowledge into the Kubernetes system, which makes Nebula Graph a real cloud-native graph database. For more information, see [What is Nebula Operator](../nebula-operator/1.introduction-to-nebula-operator.md).
+
+|Nebula Graph 版本|Operator 版本|
```
Comment (whitewum) [inline docs-2.0/20.appendix/6.eco-tool-version.md:81]: English...

**Review by whitewum — CHANGES_REQUESTED** [review]
should be english only

**Review by whitewum — DISMISSED** [review]

**Review by foesa-yang — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**